source/shared/cpp/ObjectModel/TextInput.cpp

```diff
@@ -78,7 +78,8 @@ void TextInput::SetValue(const std::string& value)
 
 bool TextInput::GetIsMultiline() const
 {
-    return m_isMultiline;
+    // isMultiline is ignored if the TextInputStyle is password
+    return m_isMultiline && m_style != TextInputStyle::Password;
```

Comment:
I'm not sure i like the idea that GetIsMultiline will return a different value than what is serialized if you serialize the card. 

I see the charm in doing this here so the renderers don't have to think about it (and don't get a say), but if we're going to let the shared model take over this logic maybe we just clear isMultiline in this case so that the serialization will come out the same as the object model?

Reply:
that's a fair point -- I'll fix it up.

Reply:
actually, given that we don't have precedent to do this kind of thing in the shared model, I think I'll pull it for now...